e2e/src/features/manage-tokens-collectibles.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Manage tokens + collectibles
+@manage_assets
+
+  Scenario: As a user, I'd like to add tokens and collectibles to my wallet [Positive]
```

Comment:
Sorry, could you add a scenario for FA2 token?

Reply:
yes i will